intro/classes/index.html

```diff
@@ -23,65 +17,65 @@ <h2>Table of Contents</h1>
     </p>
 
     <div>
-        <h2><a id="class-intro" class="js"></a>What Are Classes and Why Do We
-            Care?</h1>
+        <h2><a id="class-intro" class="js"></a>What Are Classes and Why Do We Care?</h2>
         <p>
-            <!-- TODO: reword part about attributes and methods-->
-            Python is an object oriented programming language, which means that
-            everything in Python is an object. But that doesn't tell us what objects are.
-            The word "object" here just means a collection of data and/or
-            functionality. More
-            specifically, objects contain two types of structures: attributes
-            and methods.
-            Generally speaking, attributes are variables and methods are
-            functions. Now, why do they
-            matter and how do they relate to classes? We'll get to that.
+            Python is an object oriented programming language, which means that everything in Python is an object. But that doesn't tell us what objects are.
+            The word "object" here just means a collection of data and/or functionality. More specifically, any piece of data attached to a class is 
+            called an "attribute". Attributes can be either variables or functions, but attributes that are functions have another common name: methods. Variable attributes
+            can be called "instance variables" to distinguish them from methods, but when talking about classes with other Python programmers, it is generally
+            accepted that "attribute" usually means a variable and "method" means a function. Now, why do they matter and how do they relate to classes? We'll get to that.
```

Comment:
an instance is a very important part of understanding OOP, and you reference it a lot in the article, but I don't see you defining explicitly what an instance is. I see that you said that `1` is an instance, but nothing much more.

Reply:
I elaborated below when talking about instances